appveyor.yml

```diff
@@ -30,28 +30,51 @@ environment:
     - {COMPILER: gcc, PLATFORM: Win32, MODE: Run, GRAPHICS: Direct3D9, AUDIO: None, COLLISION: None, NETWORK: None, WIDGETS: None, EXTENSIONS: "None"}
     - {COMPILER: gcc, PLATFORM: Win32, MODE: Debug, GRAPHICS: Direct3D9, AUDIO: None, COLLISION: None, NETWORK: None, WIDGETS: None, EXTENSIONS: "None"}
     - {COMPILER: gcc, PLATFORM: Win32, MODE: Compile, GRAPHICS: Direct3D9, AUDIO: None, COLLISION: None, NETWORK: None, WIDGETS: None, EXTENSIONS: "None"}
+
     # Platforms
-    - {COMPILER: gcc, PLATFORM: SDL, MODE: Debug, GRAPHICS: Direct3D9, AUDIO: None, COLLISION: None, NETWORK: None, WIDGETS: None, EXTENSIONS: "None"}
+    - {COMPILER: gcc, PLATFORM: SDL, MODE: Debug, GRAPHICS: Direct3D9, AUDIO: None, COLLISION: None, NETWORK: None, WIDGETS: None, EXTENSIONS: "None",
+       PACKAGES: mingw-w64-x86_64-SDL2}
```

Comment:
Why did you put "PACKAGES" on its own line?

Reply:
Because these lines are too fat plus PACKAGES is unlike the other variables which actually switch ENIGMA systems/extensions.

Reply:
We should probably break the lines at 80 or 100 characters if we're going to break them anywhere. But I won't press the issue right now.

Reply:
Sure, it really doesn't matter. I feel like I would personally hate looking at this file if we did that though. Not sure.